tutorials/grids.ipynb

```diff
@@ -0,0 +1,522 @@
+{
```

Comment:
<p>Suggested additional sentence: The representation of grid geometry and topology is the same for Cartesian and simplex grids. This allows for setting up simulations, defining discretization methods etc., in ways that are agnostic to the problem dimension.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/878/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>